server/controllers/abstractUserController.js

```diff
@@ -47,6 +47,14 @@ const findUsersByUsernames = (usernames) =>
         }
     });
 
+// added
+const findUserByUserId = uid =>
+    AbstractUser.findOne({
+        where: {
+            uid: uid
+        }
+    })
+
```

Comment:
This function does the exact same thing as `findAbstractUser`. Please use that function and get rid of this function

Reply:
Addressed.